scripts/libsimple_crontab

```diff
@@ -15,6 +15,9 @@ HOME=/var/www/circulation
 */15 * * * * root core/bin/run overdrive_monitor_full >> /var/log/cron.log 2>&1
 */15 * * * * root core/bin/run overdrive_monitor_recent >> /var/log/cron.log 2>&1
 */15 * * * * root core/bin/run overdrive_reaper >> /var/log/cron.log 2>&1
+0 */1 * * * root core/bin/run oneclick_library_delta >> /var/log/cron.log 2>&1
+0 0 */1 * * root core/bin/run oneclick_library_import >> /var/log/cron.log 2>&1
```

Comment:
In the Slack channel, @daryanypl [mentioned](https://librarysimplified.slack.com/archives/G5GHWH7BQ/p1498663435736053) that the import script only needs to be run once, so you can remove it from the crontab. It should be run manually, instead.